README.md

```diff
@@ -1 +1,5 @@
-# cloud-foundation-fabric
\ No newline at end of file
+# Cloud Foundation Toolkit - Fabric
+
+The Cloud Foundation Fabric provides end-to-end Terraform code examples for prototyping on GCP. Code from this repository can be used as the basis for starting to develop an infrastructure as code repository for GCP before adopting production-grade Cloud Foundation Toolkit modules.
+
+The Fabric repository is designed to be adapted and updated for your use cases. For production use cases infrastructure, alternative Cloud Foundation Toolkit modules should be considered or integrated.
```

Comment:
Not entirely clear what the alternative should be. What if we rephrase, pivoting the phrase on support instead of production-grade? Something like: "For full-featured modules with extended support...".

Fabris *is* used in production after all, with bootstrap and infra environments used mostly as-is to create the bare foundations.